taca/cleanup/cleanup.py

```diff
@@ -0,0 +1,644 @@
+"""Storage methods and utilities"""
+import getpass
+import logging
+import os
+import re
+import shutil
+import time
+
+from collections import defaultdict
+from datetime import datetime
+from glob import glob
+from multiprocessing import Pool
+
+from statusdb.db import connections as statusdb
```

Comment:
when running it I obtained this run time error:
```
IOError: There was a problem loading the configuration file. Please make sure that /Users/vezzi/.ngi_config/statusdb.yaml can be opened
```

we already have the statusdb information (password, username and link) in taca.yaml so you can take it from there. I think that this simplifies life to @sylvinite during deployment